Sources/LocalizedStringKit/include/LocalizedStringKit.h

```diff
@@ -12,7 +12,15 @@ FOUNDATION_EXPORT double LocalizedStringKitVersionNumber;
 //! Project version string for LocalizedStringKit.
 FOUNDATION_EXPORT const unsigned char LocalizedStringKitVersionString[];
 
-/// Primary localization function used to localize strings (excluding tableName)
+/// The name to be used for the primary strings bundle
+/// Example:
+/// "Localizable" (do not include the extension suffix)
+FOUNDATION_EXPORT NSString *_Nonnull primaryBundleName;
```

Comment:
We probably want to prefix this with `LSK` (for LocalizedStringsKit), otherwise this variable is in the global namespace as `primaryBundleName` and there could be a collision with a variable named `primaryBundleName`.

```suggestion
FOUNDATION_EXPORT NSString *_Nonnull LSKPrimaryBundleName;
```